### ISSUE TITLE
missing module "histcnn.argparsefuncs"

### ISSUE THREAD

**Issue by zhenzhenyang-psu** [javadnoorb/HistCNN#1]
Hello there,
I happened to find HistCNN which may help me solve my problem.
I followed the instructions in the "readme.md" and installed histcnn using "pip install ." -No error arose. 
however, while I ran "histcnn", the following error occurred:

ThanksTraceback (most recent call last):
  File "/public/home/yangzhzh/miniconda3/bin/histcnn", line 4, in <module>
    from histcnn.argparsefuncs import *
ModuleNotFoundError: No module named 'histcnn.argparsefuncs'
I looked into the folder in which the functions were placed:
/public/home/yangzhzh/miniconda3/lib/python3.7/site-packages/histcnn
and found no argparsefuncs.py script.

Is something missing from the software?

Thanks a lot!
Zhenzhen

**Comment by zhenzhenyang-psu** [comment]
hello, there,
I also ran into the following error:

File "/public/home/yangzhzh/miniconda3/bin/histcnn", line 24, in <module>
    gcs_parser.set_defaults(func=checkpath)
NameError: name 'checkpath' is not defined

Is the software just underdevelopment :P and not ready for use?
Thanks,
zhenzhen

**Comment by javadnoorb** [comment]
Hi @zhenzhenyang-psu. The repo was just used by a few of us and hasn't been thoroughly tested. So for a while I'm expecting there will be issues until we fix them. 

Regarding the issue you mentioned I have only tried this in development mode. setuptools has some quirk regarding how it reads folder names in development mode. I'm going to fix that. In the meantime, try the development mode with:

`pip install -e .` 

and to test just run:
`histcnn --help`

I tried this in a clean environment and worked. Some of the stuff have been deprecated so you'll get a bunch of warnings, but I would guess no error. I'll need to either set a fixed tensorflow version in the setup or upgrade the command to the latest version. In either case these are just warnings.

Let me know if you run into other issues.
